{"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"}]}]}],"sections":[],"variants":[{"paths":["\/documentation\/networking\/networkerror\/unknown"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"An indication of an unknown cause."}],"hierarchy":{"paths":[["doc:\/\/Networking\/documentation\/Networking","doc:\/\/Networking\/documentation\/Networking\/NetworkError"]]},"schemaVersion":{"minor":3,"major":0,"patch":0},"metadata":{"role":"symbol","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}],"modules":[{"name":"Networking"}],"externalID":"s:10Networking12NetworkErrorO7unknownyA2CmF","title":"NetworkError.unknown","symbolKind":"case"},"identifier":{"url":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/unknown","interfaceLanguage":"swift"},"kind":"symbol","references":{"doc://Networking/documentation/Networking/Response":{"kind":"symbol","role":"symbol","navigatorTitle":[{"text":"Response","kind":"identifier"}],"title":"Response","identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","type":"topic","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"Response"}],"url":"\/documentation\/networking\/response","abstract":[{"type":"text","text":"A default "},{"code":"URLSession","type":"codeVoice"},{"type":"text","text":" response tuple."}]},"doc://Networking/documentation/Networking/NetworkError":{"abstract":[{"type":"text","text":"An error that occurs during "},{"identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","isActive":true,"type":"reference"},{"text":" processing or an underlaying ","type":"text"},{"isActive":true,"identifier":"doc:\/\/Networking\/documentation\/Networking","type":"reference"},{"type":"text","text":" error."}],"url":"\/documentation\/networking\/networkerror","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError","title":"NetworkError","type":"topic","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NetworkError"}],"navigatorTitle":[{"text":"NetworkError","kind":"identifier"}]},"doc://Networking/documentation/Networking/NetworkError/unknown":{"abstract":[{"text":"An indication of an unknown cause.","type":"text"}],"url":"\/documentation\/networking\/networkerror\/unknown","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/unknown","title":"NetworkError.unknown","type":"topic","kind":"symbol","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}]},"doc://Networking/documentation/Networking":{"type":"topic","abstract":[],"url":"\/documentation\/networking","kind":"symbol","role":"collection","identifier":"doc:\/\/Networking\/documentation\/Networking","title":"Networking"}}}